elements/pfe-avatar/src/pfe-avatar.js

```diff
@@ -7,6 +7,48 @@ class PfeAvatar extends PFElement {
     return "pfe-avatar";
   }
 
+  static get properties() {
+    return {
+      name: {
+        title: "Username",
+        type: String,
+        observer: "_updateWhenConnected",
+        default: "",
+        alias: "pfeName"
+      },
+      src: {
+        title: "Avatar image URL",
+        type: String,
+        observer: "_updateWhenConnected",
+        alias: "pfeSrc"
+      },
+      pattern: {
+        title: "Shape Pattern",
```

Comment:
```suggestion
        title: "Shape pattern",
```

I mean this is super nit-picky but titles are sentence case in the writing for Red Hat guidelines 😉

Reply:
```css
*:first-letter {
    text-transform: uppercase !important;
}
```

:shrug: